ansible/configs/ocp-clientvm/files/cloud_providers/ec2_cloud_template.j2

```diff
@@ -0,0 +1,247 @@
+    ap-southeast-1:
+      RHELAMI: ami-10bb2373
+    ap-southeast-2:
+      RHELAMI: ami-ccecf5af
+    sa-east-1:
+      RHELAMI: ami-a789ffcb
+    ap-south-1:
+      RHELAMI: ami-cdbdd7a2
+  DNSMapping:
+    us-east-1:
+      domain: "us-east-1.compute.internal"
+    us-west-1:
+      domain: "us-west-1.compute.internal"
+    us-west-2:
+      domain: "us-west-2.compute.internal"
+    eu-west-1:
+      domain: "eu-west-1.compute.internal"
+    eu-central-1:
+      domain: "eu-central-1.compute.internal"
+    ap-northeast-1:
+      domain: "ap-northeast-1.compute.internal"
+    ap-northeast-2:
+      domain: "ap-northeast-2.compute.internal"
+    ap-southeast-1:
+      domain: "ap-southeast-1.compute.internal"
+    ap-southeast-2:
+      domain: "ap-southeast-2.compute.internal"
+    sa-east-1:
+      domain: "sa-east-1.compute.internal"
+    ap-south-1:
+      domain: "ap-south-1.compute.internal"
+
+Resources:
+  Vpc:
+    Type: "AWS::EC2::VPC"
+    Properties:
+      CidrBlock: "192.199.0.0/16"
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      Tags:
+        - Key: Name
+          Value: "{{vpcid_name_tag}}"
+        - Key: Hostlication
+          Value:
+            Ref: "AWS::StackId"
+
+  VpcInternetGateway:
+    Type: "AWS::EC2::InternetGateway"
+
+  VpcGA:
+    Type: "AWS::EC2::VPCGatewayAttachment"
+    Properties:
+      InternetGatewayId:
+        Ref: VpcInternetGateway
+      VpcId:
+        Ref: Vpc
+
+  VpcRouteTable:
+    Type: "AWS::EC2::RouteTable"
+    Properties:
+      VpcId:
+        Ref: Vpc
+
+  VPCRouteInternetGateway:
+    DependsOn: VpcGA
+    Type: "AWS::EC2::Route"
+    Properties:
+      GatewayId:
+        Ref: VpcInternetGateway
+      DestinationCidrBlock: "0.0.0.0/0"
+      RouteTableId:
+        Ref: VpcRouteTable
+
+  PublicSubnet:
+    Type: "AWS::EC2::Subnet"
+    DependsOn:
+      - Vpc
+    Properties:
+      CidrBlock: "192.199.0.0/24"
+      Tags:
+        - Key: Name
+          Value: "{{project_tag}}"
+        - Key: Hostlication
+          Value:
+            Ref: "AWS::StackId"
+      MapPublicIpOnLaunch: true
+      VpcId:
+        Ref: Vpc
+
+  PublicSubnetRTA:
+    Type: "AWS::EC2::SubnetRouteTableAssociation"
+    Properties:
+      RouteTableId:
+        Ref: VpcRouteTable
+      SubnetId:
+        Ref: PublicSubnet
+
+  HostSG:
+    Type: "AWS::EC2::SecurityGroup"
+    Properties:
+      GroupDescription: Host
+      VpcId:
+        Ref: Vpc
+      Tags:
+        - Key: Name
+          Value: host_sg
+
+  HostUDPPorts:
+    Type: "AWS::EC2::SecurityGroupIngress"
+    Properties:
+      GroupId:
+        Fn::GetAtt:
+          - HostSG
+          - GroupId
+      IpProtocol: udp
+      FromPort: 0
+      ToPort: 65535
+      CidrIp: "0.0.0.0/0"
+
+  HostTCPPorts:
+    Type: "AWS::EC2::SecurityGroupIngress"
+    Properties:
+      GroupId:
+        Fn::GetAtt:
+          - HostSG
+          - GroupId
+      IpProtocol: tcp
+      FromPort: 0
+      ToPort: 65535
```

Comment:
we only need port 22, no ?

Reply:
Right. I'll change that.

Do we need the internal DNS zone for just one VM? I don't think so but am not sure what parts of the Cloudformation Template I can remove...